### ISSUE TITLE
Controlbar automatically collapses when selectize input inside is changed with remove_button

### ISSUE THREAD

**Issue by tobiasblasberg** [RinteRface/bs4Dash#165]
The controlbar unfortunately automatically collapses when an option is removed with the `remove_button`. Adding an option regularly in the selectize input works. 

Here the UI code of the controlbar (!inside a module with separate NS):

``` bs4Dash::dashboardControlbar(
      id = ns("dashboardControlbar"),
      collapsed = TRUE,
      overlay = TRUE,
      controlbarMenu(id = ns("controlbarMenu"),
        controlbarItem(
          title = "Filters:",
          column(width = 12,
          selectizeInput(ns("categories_selection"), "Select Categories:",
                         choices = c("A", "B", "C"),
                         selected = c("A", "B", "C"),
                         multiple = TRUE,
                         options = list(
                           placeholder = "All Categories",
                           plugins = list("remove_button")
                         )
        ),```

**Comment by tobiasblasberg** [comment]
Same is the case for dateRangeInput

**Comment by DivadNojnarg** [comment]
Have you tried to specify the `pinned = TRUE` option inside `dashboardControlbar`? In theory it blocks the controlbar until you click on the pin icon:

```r
library(shiny)
library(bs4Dash)

shinyApp(
  ui = dashboardPage(
    header = dashboardHeader(),
    sidebar = dashboardSidebar(),
    body = dashboardBody(), controlbar = dashboardControlbar(
      id = "dashboardControlbar", 
      pinned = TRUE,
      collapsed = FALSE,
      overlay = TRUE,
      controlbarMenu(
        id = "controlbarMenu",
        controlbarItem(
          title = "Filters:",
          column(
            width = 12,
            selectizeInput("categories_selection", "Select Categories:",
              choices = c("A", "B", "C"),
              selected = c("A", "B", "C"),
              multiple = TRUE,
              options = list(
                placeholder = "All Categories",
                plugins = list("remove_button")
              )
            )
          )
        )
      )
    ),
    title = "DashboardPage"
  ),
  server = function(input, output) { }
)
```

**Comment by tobiasblasberg** [comment]
I did, but this is actually really inconvenient and not a real solution for me... 

For the meantime, I removed the "remove" plugin, which triggers the JS code and therefore an action that triggers the sidebar to close. 

However, I do not think that this issue is closed with the solution `pinned = TRUE` workaround. Even if this is not a priority now, in the long run it should be possible to use a daterange Input in the controlbarMenu, right?
